src/Confluent.SchemaRegistry.Serdes.Json/JsonSerializerConfig.cs

```diff
@@ -64,13 +64,21 @@ public static class PropertyNames
             /// <summary>
             ///     Specifies whether or not the JSON serializer should use the latest subject
             ///     version for serialization.
-            ///     WARNING: There is no check that the latest schema is backwards compatible
-            ///     with the schema of the object being serialized.
```

Comment:
Maybe: "WARNING: There is no check that the latest schema is compatible with the schema of the object being serialized by default. Use the `LatestCompatibilityStrict` config property to enable this.".